test/y2storage/storage_manager_test.rb

```diff
@@ -102,6 +102,33 @@ def devicegraph_from(file_name)
       manager.staging = new_graph
       expect(manager.proposal).to be_nil
     end
+
+    context "when trying to assign staging to itself" do
+      # It used to be in bug basically running staging.copy(staging), which
```

Comment:
> It used to be in bug basically running staging.copy(staging)

Could you rephrase this? It's not very clear what is meant.

Reply:
What about: "In the past this tried to copy staging into itself, i.e. staging.copy(staging), which caused it to become a completely empty devicegraph"

Reply:
Why not just:

In the past, copying staging into itself, i.e. staging.copy(staging), caused ...